tests/cupy_tests/statics_tests/test_meanvar.py

```diff
@@ -181,6 +181,74 @@ def test_external_std_axis_ddof(self, xp, dtype):
         return xp.std(a, axis=1, ddof=1)
 
 
+@testing.parameterize(
+    *testing.product({
+        'shape': [(3, 4), (30, 40, 50)],
+        'axis': [None, 0, 1],
+        'keepdims': [True, False]
+    })
+)
+@testing.gpu
+class TestNanMean(unittest.TestCase):
+
+    @testing.for_all_dtypes(no_float16=True)
+    @testing.numpy_cupy_allclose(rtol=1e-6)
+    def test_nanmean_without_nan(self, xp, dtype):
+        a = testing.shaped_random(self.shape, xp, dtype)
+        return xp.nanmean(a, axis=self.axis, keepdims=self.keepdims)
+
+    @testing.for_all_dtypes(no_float16=True)
+    @testing.numpy_cupy_allclose(rtol=1e-6)
+    def test_nanmean_with_nan_float(self, xp, dtype):
+        a = testing.shaped_random(self.shape, xp, dtype)
+
+        if a.dtype.kind not in 'biu':
+            a[1, :] = xp.nan
+            a[:, 3] = xp.nan
+
+        return xp.nanmean(a, axis=self.axis, keepdims=self.keepdims)
```

Comment:
Could you ignore NumPy's `RuntimeWarning` here? (I should have said about it in #2344.)

Reply:
Done. Also for `cupy.nanvar` and `cupy.nanstd` tests.